vue-frontend/src/components/android-app-development/DevelopmentSection.vue

```diff
@@ -18,7 +15,78 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+    <!-- Desktop UI -->
+    <div class="sticky-parent tw-hidden md:tw-block">
+      <div class="sticky tw-sticky tw-top-0 tw-max-h-screen tw-overflow-hidden">
+        <div class="tw-flex">
+          <div
+            v-for="(item, index) in items"
```

Comment:
last box is not stopping in center
![Screenshot from 2023-09-08 12-24-25](https://github.com/canopas/canopas-website/assets/85920585/dba0fc48-235f-4e80-9f38-f00369eae32f)